presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -358,15 +358,6 @@ public void testWithMissingColumnInOrderBy()
         assertOptimizedQuery(baseQuerySql, baseQuerySql, originalViewSql, BASE_TABLE_1, VIEW_1);
     }
 
-    @Test
```

Comment:
Why did we remove this?

Reply:
I removed where we check limit from `MaterializedViewQueryOptimizer` to `MaterializedViewQueryShapeValidator`. With these changes, it's no longer `Optimizer`'s job to do this check as it's able to be done without access to the MV itself.

Reply:
I got confused about which query we were parsing and thought I could add this check to the new validator- I cannot, thanks for catching this or I would have been shipping a bug.